docs/verify/simulate-with-qiskit-aer.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<p>when you introduce Qiskit Aer can you emphasise that it is a Qiskit Ecosystem project, and link to <a href="https://qiskit.github.io/ecosystem/" target="_blank">https://qiskit.github.io/ecosystem/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done.